wiki/markdown/add-subscription.md

```diff
@@ -49,7 +49,7 @@ used as a password for authentication against the entire jenkins instance.
 **Authentication:** A list to select authentication type.
```

Comment:
Looks good! 
I was thinking, you could also change each paragraph to have it's own header. That way we can link directly to a specific section on this page. For example, from the back-end we want to link to the Authentication section of this page and not the entire file.

Reply:
fixed.